pkg/kubelet/kubelet_linux.go

```diff
@@ -49,3 +56,52 @@ func (kl *Kubelet) cgroupVersionCheck() error {
 	}
 	return nil
 }
+
+func (kl *Kubelet) startHealthCheck() error {
+	if runtime.GOOS != "linux" {
+		return fmt.Errorf("systemd watchdog can only be used on linux")
+	}
+
+	watchdogVal, err := daemon.SdWatchdogEnabled(false)
+	if err != nil {
+		return fmt.Errorf("could not check if systemd watchdog information: %v", err)
+	}
+	if watchdogVal == 0 {
+		return fmt.Errorf("no watchdog interval is configured")
+	}
+
+	// Periodically perform health checks and notify systemd.
+	kl.checkAndNotify(watchdogVal / 2)
```

Comment:
let's log the interval value in the log message so it is easier to troubleshoot if something will go wrong